pandas/tests/series/test_constructors.py

```diff
@@ -53,6 +53,12 @@
 
 
 class TestSeriesConstructors:
+    def test_from_na_value_and_interval_of_datetime_dtype(self):
```

Comment:
Need a whatsnew?

Reply:
i dont think so. this just adds the test, the bug got fixed sometime previously